client/my-sites/plans-v2/translated-copy.tsx

```diff
@@ -0,0 +1,203 @@
+			return {
+				displayName: translate( 'Backup {{em}}Real-Time{{/em}}', {
+					components: {
+						em: <em />,
+					},
+				} ),
+				shortName: translate( 'Backup Real-time', {
+					comment: 'Short name of the Jetpack Backup Real-time product',
+				} ),
+				tagline: translate( 'Best for sites with frequent updates' ),
+				description: translate(
+					'Real-time backups save every change and one-click restores get you back online quickly.'
+				),
+				buttonLabel: translate( 'Get Backup {{em}}Real-Time{{/em}}', {
+					components: {
+						em: <em />,
+					},
+				} ),
+			};
+		case PLAN_JETPACK_COMPLETE:
+		case PLAN_JETPACK_COMPLETE_MONTHLY:
+			return {
+				displayName: translate( 'Jetpack Complete' ),
+				shortName: translate( 'Complete', {
+					comment: 'Short name of Jetpack Complete',
+				} ),
+				tagline: translate( 'For best-in-class WordPress sites' ),
+				description: translate(
+					'Superpower your site with everything Jetpack has to offer: real-time security, enhanced search, CRM, and marketing, growth, and design tools.'
+				),
+				buttonLabel: translate( 'Get Jetpack Complete' ),
+			};
+		case PRODUCT_JETPACK_CRM:
+		case PRODUCT_JETPACK_CRM_MONTHLY:
+			return {
+				displayName: translate( 'Jetpack CRM' ),
+				shortName: translate( 'CRM', {
+					comment: 'Short name of the Jetpack CRM',
+				} ),
+				tagline: translate( 'Manage contacts effortlessly' ),
+				description: translate(
+					'The most simple and powerful WordPress CRM. Improve customer relationships and increase profits.'
+				),
+				buttonLabel: translate( 'Get CRM' ),
+			};
+		case PRODUCT_JETPACK_SCAN:
+		case PRODUCT_JETPACK_SCAN_MONTHLY:
+			return {
+				displayName: translate( 'Jetpack Scan' ),
+				shortName: translate( 'Scan', {
+					comment: 'Short name of Jetpack Scan',
+				} ),
+				tagline: translate( 'Protect your site' ),
+				description: translate(
+					'Automatic scanning and one-click fixes keep your site one step ahead of security threats.'
+				),
+				buttonLabel: translate( 'Get Scan' ),
+			};
+		case OPTIONS_JETPACK_SECURITY:
+		case OPTIONS_JETPACK_SECURITY_MONTHLY:
+			return {
+				displayName: translate( 'Jetpack Security' ),
+				shortName: translate( 'Security', {
+					comment: 'Short name of the Jetpack Security generic plan',
+				} ),
+				tagline: translate( 'Comprehensive WordPress protection' ),
+				description: translate(
+					'Enjoy the peace of mind of complete site security. ' +
+						'Easy-to-use, powerful security tools guard your site, so you can focus on your business. ' +
+						'{{em}}Available options: Real-time or Daily.{{/em}}',
+					{ components: { em: <em /> } }
+				),
+				buttonLabel: translate( 'Get Jetpack Security' ),
+			};
+		case PLAN_JETPACK_SECURITY_DAILY:
+		case PLAN_JETPACK_SECURITY_DAILY_MONTHLY:
+			return {
+				displayName: translate( 'Security {{em}}Daily{{/em}}', { components: { em: <em /> } } ),
+				shortName: translate( 'Security Daily', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Security Daily', { context: 'Jetpack Security Daily Plan'} )` *ES Score: 15*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/move-plansv2-translates-from-constants-take-2)